apipie-bindings.gemspec

```diff
@@ -34,5 +36,5 @@ EOF
   s.add_development_dependency 'mocha'
   s.add_development_dependency 'ci_reporter', '>= 1.6.3', "< 2.0.0"
```

Comment:
I think this is actually unused. We typically used it to generate junit output in Jenkins but that isn't used in GHA.